flixel/FlxGame.hx

```diff
@@ -243,6 +243,14 @@ class FlxGame extends Sprite
 		// Super high priority init stuff
 		_inputContainer = new Sprite();
 		
+		if (GameSizeX <= 0)
+		{
+			GameSizeX = Std.int(Lib.current.stage.window.width);
```

Comment:
I have tried using `Lib.current.stage.width` but not `Lib.current.stage.*window*.width`.
BUT, `Lib.current.stage.width` **IS 0!** I don't know why.
I figured that out when targeting flash.